Content.Shared/Alert/AlertsSystem.cs

```diff
@@ -74,7 +76,8 @@ public bool TryGetAlertState(EntityUid euid, AlertKey key, out AlertState alertS
     /// <param name="severity">severity, if supported by the alert</param>
     /// <param name="cooldown">cooldown start and end, if null there will be no cooldown (and it will
     ///     be erased if there is currently a cooldown for the alert)</param>
-    public void ShowAlert(EntityUid euid, AlertType alertType, short? severity = null, (TimeSpan, TimeSpan)? cooldown = null)
+    /// /// <param name="autoRemove">the time when the alert will be removed automatically</param>
+    public void ShowAlert(EntityUid euid, AlertType alertType, short? severity = null, (TimeSpan, TimeSpan)? cooldown = null, TimeSpan? autoRemove = null)
```

Comment:
"the time when the alert will be removed" implies it's the server time when it's removed not the delay, probably call it "removeAfter" instead.

Reply:
It is indeed the server time, but I made the wording more specific